mappings/net/minecraft/item/Item.mapping

```diff
@@ -139,7 +144,9 @@ CLASS net/minecraft/class_124 net/minecraft/item/Item
 		ARG 7 side
 	METHOD method_446 postHit (Lnet/minecraft/class_31;Lnet/minecraft/class_127;Lnet/minecraft/class_127;)Z
 		ARG 1 stack
-	METHOD method_447 getDamageOnEntity (Lnet/minecraft/class_57;)I
+		ARG 2 target
+		ARG 3 attacker
+	METHOD method_447 getAttackDamage (Lnet/minecraft/class_57;)I
 		ARG 1 entity
 	METHOD method_448 getIsHandheld ()Z
```

Comment:
The "get" prefix breaks the naming standard in this case. Boolean methods used for checking something do not have this prefix anywhere else.

Reply:
renamed to isHandheld